zeppelin-interpreter/src/main/java/org/apache/zeppelin/interpreter/remote/RemoteInterpreterUtils.java

```diff
@@ -58,21 +65,22 @@ public static int findRandomAvailablePortOnAllLocalInterfaces() throws IOExcepti
    * @return
    * @throws IOException
    */
-  public static int findRandomAvailablePortOnAllLocalInterfaces(String portRange)
+  public static TServerSocket createTServerSocket(String portRange)
       throws IOException {
 
+    TServerSocket tSocket = null;
     // ':' is the default value which means no constraints on the portRange
     if (portRange == null || portRange.equals(":")) {
```

Comment:
what if `portRange` == `""`?

Reply:
Recently, I'm using `StringUtils.isBlank` which check null, empty and whitespaces only

Reply:
Fixed